nominal/_tdms.py

```diff
@@ -27,7 +27,10 @@ def tdms_with_time_column_to_pandas(path: Path, timestamp_column: str) -> pd.Dat
 
     # format for nominal upload
     df = functools.reduce(
-        lambda left, right: pd.merge(left, right, left_index=True, right_index=True, how="outer"), group_dfs
+        lambda left, right: pd.merge(
+            left, right, left_index=True, right_index=True, how="outer", suffixes=["_repeated_1, _repeated_2"]
```

Comment:
is this supposed to be `suffixes=["_repeated_1", "_repeated_2"]`?

Reply:
Yes that is a good catch. Will update.

Reply:
Resolved in latest commit